.github/workflows/jars-publish.yaml

```diff
@@ -0,0 +1,48 @@
+name: Build kotlin artifacts and publish to github packages
```

Comment:
@jagpreetsinghsasan Sorry, some nit-pick here. We might have more kotlin artifacts for other connectors in the future so let's not crowd out everyone else by not specifying that this is the corda API client.

TLDR: Name the job Something like `Corda_API_Client` instead of `Build kotlin artifacts and publish to github packages`

Reply:
@jagpreetsinghsasan Please also rename the file in the same spirit so that it's less ambiguous once we have more jobs publishing more jars. 
Recommended filename: `corda-api-client-publish.yaml`

Reply:
@petermetz I have also coded for the bigger task where the kotlin client is generated at 22 more places.
![image](https://user-images.githubusercontent.com/56861721/179166290-a4329a75-c559-4be0-99f9-6bba8729454f.png)

Are we expecting 22 different github workflows ?(I generalised it so that we can different steps for different openapi files instead of having different workflows altogether)

Reply:
@petermetz any updates on this?

Reply:
@jagpreetsinghsasan Sorry for the slow response. I'm not sure TBH => If the tasks are long running (more than a few minutes) then I would break them up into separate workflow action *jobs* at the very least (e.g. they could stay in the same file but at least have separate jobs within).
So we don't necessarily need 22 workflows (and then we might not need to name the file so specifically either)

> @petermetz I have also coded for the bigger task where the kotlin client is generated at 22 more places.

Could you send a link to the version of the file that has the 22 more places and then I can provide a definitive answer!

Reply:
I am not able to provide a link for the parent issue as there is an issue with the `openapi-generator-cli`
The latest version of the cli uses `openapi-generator` version 5.2.0 but in that there is a bug where the type `any` is rendered as `AnyType`. This bug got fixed in 5.2.1 version of `openapi-generator` which isnt currently supported by `openapi-generator-cli` (I have linked detailed information in the personal chat @petermetz )

Reply:
The parent code is also ready, just stuck there because of the above blocker.

Reply:
> @jagpreetsinghsasan Sorry for the slow response. I'm not sure TBH => If the tasks are long running (more than a few minutes) then I would break them up into separate workflow action _jobs_ at the very least (e.g. they could stay in the same file but at least have separate jobs within). So we don't necessarily need 22 workflows (and then we might not need to name the file so specifically either)
> 
> > @petermetz I have also coded for the bigger task where the kotlin client is generated at 22 more places.
> 
> Could you send a link to the version of the file that has the 22 more places and then I can provide a definitive answer!

Hi @petermetz . Here is a [link](https://github.com/jagpreetsinghsasan/cactus/actions/runs/3383545104/jobs/5619542626) to such workflow (please ignore the errors, need to see why some of them are still failing).

Reply:
> > @jagpreetsinghsasan Sorry for the slow response. I'm not sure TBH => If the tasks are long running (more than a few minutes) then I would break them up into separate workflow action _jobs_ at the very least (e.g. they could stay in the same file but at least have separate jobs within). So we don't necessarily need 22 workflows (and then we might not need to name the file so specifically either)
> > > @petermetz I have also coded for the bigger task where the kotlin client is generated at 22 more places.
> > 
> > 
> > Could you send a link to the version of the file that has the 22 more places and then I can provide a definitive answer!
> 
> Hi @petermetz . Here is a [link](https://github.com/jagpreetsinghsasan/cactus/actions/runs/3383545104/jobs/5619542626) to such workflow (please ignore the errors, need to see why some of them are still failing). And here's the [link](https://github.com/jagpreetsinghsasan/cactus/blob/feature-468/.github/workflows/jars-publish.yaml) to the corresponding code

@jagpreetsinghsasan The kotlin build runs for 30 seconds while the configure script takes 6 minutes so it makes sense to combine all the kotlin builds into one workflow so that they can share the configure script's results instead of having a bunch of only slightly smaller workflows where each of them runs the configure script for the majority of their execution time.

Also, as usual, sorry for the slow response!

Reply:
Sure. This seems to be pretty optimised approach.